css/main.scss

```diff
@@ -136,6 +145,33 @@ header {
 }
 .footer {margin-top:40px;}
 
+#mobileFooter {
+  display: none;
+}
+
+@media screen and (max-width: 768px){
```

Comment:
this is good but something to keep in mind is that if the width doesn't go from end to end, there may be cases where the right side is exposed. (see where the city and the footer interfere with each other)

<img width="487" alt="Screen Shot 2020-06-11 at 7 29 50 PM" src="https://user-images.githubusercontent.com/1668411/84452047-5d242b00-ac1a-11ea-83f6-10478b1240d5.png">

Reply:
+1

Reply:
+1